src/toolbar/Toolbar.jsx

```diff
@@ -1,15 +1,256 @@
 var React = require('react');
+var ReactDOM = require("react-dom");
+var FontIcon = require("../FontIcon.jsx");
+var ToolbarGroup = require("../toolbar/ToolbarGroup.jsx");
+
+/**
+ * This variable will hold all the icons to be displayed in the toolbar if we are in mobile mode display
+ *
+ * @type {Array}
+ */
+var toolbarIcons = [];
+
+/**
+ * The default icon width that we will use as our reference
+ *
+ * @constant
+ */
+var DEFAULTICONWIDTH = 48;
```

Comment:
This makes me uneasy since it could create an unexpected bug if any other themes or anyone ever changes the default icon width.